integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonAnalysisRules.scala

```diff
@@ -170,6 +171,7 @@ object CarbonIUDAnalysisRule extends Rule[LogicalPlan] {
     logicalplan transform {
       case UpdateTable(t, cols, sel, where) => processUpdateQuery(t, cols, sel, where)
       case DeleteRecords(statement, table) => processDeleteRecordsQuery(statement, table)
+      case ShowPartitions(t) => ShowCarbonPartitionsCommand(t)
```

Comment:
This should not be here, handle this in DDLStrategy. 
Spark2 already parses the "show partition" command for us so we can match the same in DDLStrategy.